third_party/2and3/click/core.pyi

```diff
@@ -45,47 +46,47 @@ def iter_params_for_processing(
 class Context:
     parent: Optional['Context']
     command: 'Command'
-    info_name: Optional[str]
+    info_name: Optional[Text]
     params: Dict
-    args: List[str]
-    protected_args: List[str]
+    args: List[Text]
+    protected_args: List[Text]
     obj: Any
-    default_map: Mapping[str, Any]
-    invoked_subcommand: Optional[str]
+    default_map: Mapping[Text, Any]
+    invoked_subcommand: Optional[Text]
     terminal_width: Optional[int]
     max_content_width: Optional[int]
     allow_extra_args: bool
     allow_interspersed_args: bool
     ignore_unknown_options: bool
-    help_option_names: List[str]
-    token_normalize_func: Optional[Callable[[str], str]]
+    help_option_names: List[Text]
+    token_normalize_func: Optional[Callable[[Text], Text]]
     resilient_parsing: bool
-    auto_envvar_prefix: Optional[str]
+    auto_envvar_prefix: Optional[Text]
     color: Optional[bool]
-    _meta: Dict[str, Any]
+    _meta: Dict[Text, Any]
     _close_callbacks: List
     _depth: int
 
     # properties
-    meta: Dict[str, Any]
-    command_path: str
+    meta: Dict[Text, Any]
+    command_path: Text
 
     def __init__(
         self,
         command: 'Command',
         parent: Optional['Context'] = ...,
-        info_name: Optional[str] = ...,
+        info_name: Optional[Text] = ...,
         obj: Optional[Any] = ...,
-        auto_envvar_prefix: Optional[str] = ...,
-        default_map: Optional[Mapping[str, Any]] = ...,
+        auto_envvar_prefix: Optional[Text] = ...,
+        default_map: Optional[Mapping[Text, Any]] = ...,
         terminal_width: Optional[int] = ...,
         max_content_width: Optional[int] = ...,
         resilient_parsing: bool = ...,
         allow_extra_args: Optional[bool] = ...,
         allow_interspersed_args: Optional[bool] = ...,
         ignore_unknown_options: Optional[bool] = ...,
-        help_option_names: Optional[List[str]] = ...,
-        token_normalize_func: Optional[Callable[[str], str]] = ...,
+        help_option_names: Optional[List[Text]] = ...,
```

Comment:
Can this be a Sequence or Iterable instead so that lists of strs will be accepted?